pkg/cmd/kafka/create/create.go

```diff
@@ -93,6 +93,10 @@ func NewCreateCommand(f *factory.Factory) *cobra.Command {
 				}
 			}
 
+			if opts.bypassChecks && (opts.marketplace != "" || opts.marketplaceAcctId != "") {
```

Comment:
FYI Bypass checks is basically an hack for kas-installer. We do not need to support flags like this. They can be ignored. Any billing info is not applied there